linchpin/provision/roles/aws/tasks/teardown_resource_group.yml

```diff
@@ -23,6 +23,13 @@
   when: auth_var is defined
   no_log: "{{ not auth_debug }}"
 
+- shell:
```

Comment:
@14rcole Do we need these debug statements?

Reply:
Nope, thank you